content/client-lib-development-guide/features.textile

```diff
@@ -1730,11 +1734,19 @@ class RealtimeChannel:
   publish(name: String?, data: Data?) => io // RTL6i
   subscribe((Message) ->) => io // RTL7a
   subscribe(String, (Message) ->) => io // RTL7b
+  subscribe(MessageFilterObject, (Message) ->) // RTL22
   unsubscribe() // RTL8a, RTE5
   unsubscribe((Message) ->) // RTL8a
   unsubscribe(String, (Message) ->) // RTL8a
+  unsubscribe(MessageFilterObject, (Message) ->) // RTL22
   setOptions(options: ChannelOptions) => io // RTL16
 
+class MessageFilterObject:
+  hasRef: bool // RTL22b
```

Comment:
I think `hasRef` (or `hasReference`) is a little ambiguous, since it reads as "this message has references" which could be interpreted as "other messages exist which reference this message" (i.e. it is the target of some other message which references it).

I think really, the requirement is to filter for messages which either are or are not a reference to another message, so "is" is probably better than "has" (i.e. `isRef` / `isReference`).

WDYT?

Reply:
Good point, from a users perspective isRef makes much more sense. [Done](https://github.com/ably/docs/pull/1444/commits/9bbc7c471ce1828decf3127b4a48f3275388e747).